Ginger/GingerCoreTest/Misc/DataBaseTest.cs

```diff
@@ -362,5 +362,53 @@ public void RenameDBName()
             Assert.AreEqual(actDB.DBName, db.Name, "Names");
 
         }
+
+        [TestMethod]
+        [Timeout(60000)]
+        public void TestHidePasswordFromString()
+        {
+            //Access Db
+            string ConString = "Provider=Microsoft.ACE.OLEDB.12.0;Data Source=C:/myAccessFile.accdb;Jet OLEDB:Database Password=MyDbPassword;";
+            string Result = General.HidePasswordFromString(ConString);
+            Assert.AreNotEqual(ConString, Result);
+            Assert.AreEqual(Result.Contains("Password=*****"), true);
```

Comment:
for all Asserts, please add also assert which make sure the actual password was removed, like:
Assert.AreEqual(Result.Contains("MyDbPassword"), false);